src/main/kotlin/no/nav/familie/ef/iverksett/oppgave/OppgaveService.kt

```diff
@@ -97,7 +98,7 @@ class OppgaveService(
         if (forrigeBehandling !is IverksettOvergangsstønad) {
             error("Forrige behandling er av annen type=${forrigeBehandling::class.java.simpleName}")
         }
-        if (forrigeBehandling.erMigrering()) {
+        if (forrigeBehandling.gjeldendeVedtak().periodeType == VedtaksperiodeType.MIGRERING) {
```

Comment:
Smidig løsning, får du testet det i dev? :)

Reply:
Denne er nå testet av oss, inkludert Mirja. For å teste dette ble det opprettet dummy migreringer, satt G-omregning-dato i tilkjent ytelse ett år tilbake for å trigge automatisk g-omregning, som igjen kunne gjøres revurderinger på.